internal/storage/upgrades.go

```diff
@@ -0,0 +1,77 @@
+package storage
+
+import (
+	"database/sql"
+	"fmt"
+
+	"github.com/authelia/authelia/internal/utils"
+)
+
+func (p *SQLProvider) upgradeSchemaVersionTo001(tx *sql.Tx, tables []string) error {
+	_, err := tx.Exec(p.sqlCreateConfigTable)
+	if err != nil {
+		return err
+	}
+
+	_, err = tx.Exec(p.sqlConfigSetValue, "schema", "version", "1")
+	if err != nil {
+		return err
+	}
+
+	if !utils.IsStringInSlice(preferencesTableName, tables) {
```

Comment:
what do you think about creating a map with the table name as the key and the sql statement as value and iterate over it to avoid the boilerplate?

Reply:
That should be easy to do, and a good pattern going forward.